{
  "debianbug": 844011,
  "description": "The dwarf_get_aranges_list function in dwarf_arrange.c in Libdwarf before 20161124 allows remote attackers to cause a denial of service (out-of-bounds read).",
  "releases": {
    "bullseye": {
      "fixed_version": "20161124-1",
      "repositories": {
        "bullseye": "20190529-1"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "buster": {
      "fixed_version": "20161124-1",
      "repositories": {
        "buster": "20180809-1"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "jessie": {
      "nodsa": "Minor issue",
      "nodsa_reason": "",
      "repositories": {
        "jessie": "20120410-2+deb8u1"
      },
      "status": "open",
      "urgency": "medium**"
    },
    "sid": {
      "fixed_version": "20161124-1",
      "repositories": {
        "sid": "20190529-1"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "stretch": {
      "fixed_version": "20161124-1",
      "repositories": {
        "stretch": "20161124-1+deb9u1"
      },
      "status": "resolved",
      "urgency": "medium**"
    }
  },
  "scope": "remote"
}